config/routes.rb

```diff
@@ -11,7 +11,11 @@
   devise_for :employees, path: 'employees', controllers: { sessions: "employees/sessions", registrations: "employees/registrations" }
 
   resources :kudos
-  resources :rewards, only: [:index, :show]
+  resources :rewards, only: [:index, :show] do
+    collection do
+      post 'buy_reward/:id', to: 'rewards#buy_reward', as: 'buy_reward'
```

Comment:
I would simplify the path to buying a reward:
```suggestion
      resources :orders, only: %i[index create]
```